### DISCUSSION TITLE
Automatic Relationship Expiration

### DISCUSSION THREAD

**Discussion by nkreiger** [openfga/community#177] [Q&A] [answered]
It would be really cool to be able to create relationships that automatically expire after a certain period of time. For temporary access in a distributed system

**Comment by aaguiarz** [comment, marked as answer]
Hi @nkreiger 

We plan to support that as part of this work https://github.com/openfga/roadmap/issues/32

**Reply by nkreiger** [reply]
@aaguiarz Awesome! Thanks for the heads up, I see that it is in progress, looking forward to it
